src/sync_template.sh

```diff
@@ -28,22 +28,29 @@ if [[ -n "${SRC_SSH_PRIVATEKEY_ABS_PATH}" ]]; then
   export GIT_SSH_COMMAND="ssh -i ${SRC_SSH_PRIVATEKEY_ABS_PATH}"
 fi
 
-TEMPLATE_VERSION_FILE_NAME=".templateversionrc"
-TEMPLATE_SYNC_IGNORE_FILE_NAME=".templatesyncignore"
+TEMPLATE_VERSION_FILE_PATH=".templateversionrc"
+TEMPLATE_SYNC_IGNORE_FILE_PATH=".templatesyncignore"
 TEMPLATE_REMOTE_GIT_HASH=$(git ls-remote "${SOURCE_REPO}" HEAD | awk '{print $1}')
 NEW_TEMPLATE_GIT_HASH=$(git rev-parse --short "${TEMPLATE_REMOTE_GIT_HASH}")
 NEW_BRANCH="${PR_BRANCH_NAME_PREFIX}_${NEW_TEMPLATE_GIT_HASH}"
 
 echo "::group::Check new changes"
 echo "::debug::new Git HASH ${NEW_TEMPLATE_GIT_HASH}"
-if [ -r ${TEMPLATE_VERSION_FILE_NAME} ]
-then
-  CURRENT_TEMPLATE_GIT_HASH=$(cat ${TEMPLATE_VERSION_FILE_NAME})
+
+# Check if the Version File exists inside root of the repository
+if [[ -f "$TEMPLATE_VERSION_FILE_PATH" ]]; then
```

Comment:
@LuisHenri can you  please change the search order here?
I like to prefer `.github/` before the repo root.
- first search in `.github/` folder for the file
  - if present -> use that file
  - - if not present, search in repo root

Reply:
I guess it should be fine now.
It first checks if the files are inside `.github` folder OR if they DO NOT exist in the root.
That would mean:
1. If file in `.github` -> Use it
2. If file NOT in `github`, BUT it also doesn't exist in the root -> Means it doesn't exist at all -> Use as if in `.github`.
3. Otherwise -> File in root -> Use it

Reply:
I also changed the Diagram. :)